docs/community/oidc-integrations.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: default
+title: Community-Tested OIDC Integrations
+parent: Community
+nav_order: 4
+---
+
+# OIDC Integratins
+
+**Note** This is community-based content for which the core-maintainers cannot guarantee correctness. The parameters may change over time. If a parameter does not work as documented, please submit a PR to update the list.
+
+## Currently Tested Applications
```

Comment:
Grafana 8.0.5 has been tested and works.

Reply:
added